chromium_src/BUILD.gn

```diff
@@ -599,6 +612,144 @@ source_set("browser") {
   }
 }
 
+source_set("safe_browsing") {
+  sources = [
+    "//chrome/browser/download/download_completion_blocker.cc",
```

Comment:
safe browsing has an existing target in chrome so I don't think we should have to add all these files individually